__tests__/costumes.test.js

```diff
@@ -0,0 +1,455 @@
+const {
+    createTables, 
+    createCostume, 
+    getAllCostumes, 
+    getCostumeById, 
+    updateCostume, 
+    deleteCostumeById
+} = require(".././index");
+const {Pool} = require("pg");
+
+let pool; 
+if (process.env.NODE_ENV === "test") {
+    pool = new Pool({
+        host: 'localhost',
+        port: 5432,
+        database: 'costume_shop_db_test'
+    });
+    pool.on("error", (error) => {
+        console.error(error.stack())
+    })
+} else {
+    throw new Error("NODE_ENV environment variable not set to 'test'. Testing aborted.")
+}  
+
+// Disconnect from postgres database after all tests done
+afterAll(async () => {
+    await pool.end()
+})
+
+describe("createTables adapter", () => {
+    it("should create a table", async () => {
+        //TODO: Fix this open handle
+        // console.log("connected");
+        await createTables();
+        const costumes = await pool.query(`
+            SELECT * FROM costumes;
+        `)
+        expect(costumes).toBeTruthy();
+    })
+})
+
+describe("createCostume adapter", () => {
+    it("should create a new row in the table", async () => {
+        //TODO: Fix this open handle
+        console.log("connected");
+        await createTables();
+        const {rows} = await pool.query(`
+            SELECT COUNT(*) FROM costumes;
+        `)
+        const rowsBefore = rows[0].count;
+        await createCostume(
+            "ballroom gown",
+            "adult",
+            "female",
+            "L",
+            "dress",
+            1,
+            150.99
```

Comment:
another advantage of using immediately destructured args in your functions (see my comment on the previous pull request) -would be that for these tests you could hard-code a couple of costume arg objects and store them in constants, and then re-use them as constants across all of these tests without needing to keep hard-coding things.

Reply:
I thought of this and then thought that having unique arguments for each test might ensure that data added to the database from a previous test doesn't confound the result. I suppose if the Pool API is working properly that shouldn't happen. I agree, it would be nice to streamline things and reuse some objects. I'll go forward with that approach unless you think there is merit to my old approach.